docs/notes_for_users.md

```diff
@@ -175,7 +173,56 @@ configurable options of _pydoclint_, and
 [here](https://jsh9.github.io/pydoclint/how_to_config.html) is how to configure
 _pydoclint_.
 
-## 6. How to adopt _pydoclint_ more easily in legacy projects
+## 6. Writing default arguments in numpy-style docstrings
+
+The [numpy docstring style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
+specifies a few different ways to include default values for
+function args in the the docstring.
+
+For a function with a signature like this:
+
+```python
+def some_fn(arg1: int = 10):
+```
+
+The following styles are currently supported:
+
+```python
+    Parameters
+    ----------
+    arg1 : int, default 10
+
+    arg1 : int, default is 10
+
+    arg1 : int, default: 10
+
+    arg1 : int = 10
+```
+
+And for a function with a signature like this:
+
+``python
+def some_fn(arg1: int | None = None):
+
+````
+
+The following styles are currently supported:
+```python
+    Parameters
+    ----------
+    arg1 : int, optional
+
+    arg1 : int | None, default is None
+
+    arg1 : int | None, default: None
+
+    arg1 : int | None = None
+````
+
+The portion following the type hints are ignored and not checked for
+congruence with the function signature.
```

Comment:
I think if we want to allow annotating default values in the docstring, we need to check the congruence (between default value in the docstring and the default value in the function signature).  Otherwise the behavior is surprising and implicit (i.e., we selectively check something, but not others).

Could you think of a way to also check the congruence of the default values as well?